debug/test_elevators.py

```diff
@@ -0,0 +1,70 @@
+import argparse

```

Comment:
One more util, this will go room by room and test it with every combo of elevators a set number of times and then report back how many of those combos succeeded.